src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_callcodeFiller.json

```diff
@@ -0,0 +1,76 @@
+{
+    "returndatasize_after_callcode" : {
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x00",
+                "code": "{ (seq (CALLCODE 60000 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) (SSTORE 0 (RETURNDATASIZE)))}",
+                "nonce" : "0x00",
+                "storage" : {
+                    "0x00" : "0x00"
```

Comment:
If you start with
```
"0x00" : "0xffff"
```
or something big, you can distinguish `(RETURNDATASIZE)` throwing vs. `(RETURNDATASIZE)` returning zero.

This is just a possible improvement and does not block the merge.

Reply:
Good catch.  I'll make sure to incorporate this in the implementation of remaining `RETURNDATACOPY/RETURNDATASIZE` test cases.